rachelekm/vegishphilly
# Add basic User endpoints to Django

## Overview

This PR creates basic endpoints to request User details through Django rest_framework

### Notes

Right now UserSerializer fields are only for "id" and "username" - in the future will need to incorporate more fields to better fit features (e.g. boolean field to determine if vendor or not, date_added, etc) 


## Testing Instructions

 * Check out this branch and update
 * If you have no user, `scripts/manage createsuperuser`
 * Use curl to get access token with your new user credentials:
```
 curl \
  -X POST \
  -H "Content-Type: application/json" \
  -d '{"username": "YOUR_USERNAME", "password": "YOUR_PASSWORD"}' \
  http://localhost:8033/api/token/
```
* Use access token to get user details:
```
curl \
  -H "Authorization: Bearer ACCESS_TOKEN" \
  http://localhost:8033/users/
```
* Verify endpoint works and users returned
